src/ontology/mondo-edit.obo

```diff
@@ -199962,7 +199963,7 @@ xref: UMLS:C0221406 {source="MEDGEN:66381", source="MONDO:equivalentTo", source=
 is_a: MONDO:0003429 {source="DOID:7004", source="Orphanet:96253"} ! functioning pituitary gland adenoma
 is_a: MONDO:0017824 {source="DOID:7004/inferred", source="MONDO:0009050/inferred", source="MONDO:Redundant", source="OMIM:219090", source="Orphanet:96253/inferred"} ! familial isolated pituitary adenoma
 is_a: MONDO:0020528 {source="Orphanet:96253", source="icd11.foundation:380861892"} ! ACTH-dependent Cushing syndrome
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/12631 {source="OMIM:219090"} ! USP8
+relationship: RO:0004004 http://identifiers.org/hgnc/12631 {source="OMIM:219090"} ! has material basis in somatic mutation in USP8
```

Comment:
confirming that you meant "somatic" mutation. 
If it is correct, we need to make sure that we update the other document to ensure that this annotation doesn't get updated via the automated pipelines